source/cloud-native-platform/troubleshooting/index.html.md.erb

```diff
@@ -505,3 +505,39 @@ Check the pipeline passed after merging. If the pipeline failed, it's possible t
 Check the IP your rule is forwarding to. It should be the private IP of the frontend Application Gateway.
 
 You can find this [here](https://github.com/hmcts/azure-platform-terraform/blob/6f0b867e75b7e9cee9e7adc87084f6911eb5373d/environments/sbox/sbox.tfvars#L20).
+
+### Java (Spring Boot) Golden Path Errors
+
+#### - Suppress false positives in the OWASP Dependency Checker
+
+The [OWASP Dependency Checker](https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/index.html) provides monitoring of the project's dependencies and creates a report of known
+vulnerable components that are included in the build. The checker plugin is declared in the Golden path for Java apps. Because of the way the dependency checker works, false positives and false negatives may exist. We can surepress these false positives by providing
+the dependency checker with the path to a suppression file in the `build.gradle` file.
+
+```
+dependencyCheck {
+  suppressionFile = 'path/to/supression.xml'
+}
+```
```

Comment:
Perhaps we can add that there should be an existing suppressions file which can be updated and maybe a code snippet showing an example entry